module/src/utils/arrays.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { Arrays } from './arrays';
+
+describe('Array utils', () => {
+  it('flatten - Convert an array of arrays into a single array', () => {
```

Comment:
personal preference but would the pattern be better written like this

```describe('flatten', () => {
   it('should convert an array of arrays into a single array', () => {
      ...
   }
})```

Reply:
Given the utility tests all need a proper rewrite anyway, will pick this up in that Jira ticket.